.gitignore

```diff
@@ -8,3 +8,5 @@
 /captures
 /.externalNativeBuild
 
+
+.idea/encodings.xml
```

Comment:
I don't think this will do anything since the file is committed, but I'm not really worried about it.